test/unit/check/arbitrary/helpers/ArrayHelper.spec.ts

```diff
@@ -0,0 +1,46 @@
+import * as fc from '../../../../../lib/fast-check';
+import { findOrUndefined } from '../../../../../src/check/arbitrary/helpers/ArrayHelper';
+
+describe('ArrayHelper', () => {
+  describe('findOrNull', () => {
+    it('should return null for empty array', () => {
```

Comment:
You mean `undefined`
```suggestion
    it('should return undefined for empty array', () => {
```

Reply:
Fixed